code/game/objects/items/weapons/grenades/chem_grenade.dm

```diff
@@ -238,6 +238,30 @@
 		beakers += B2
 		icon_state = initial(icon_state) +"_locked"
 
+/obj/item/weapon/grenade/chem_grenade/gas
+	name = "NT. 53 'Sandman' grenade"
+	desc = "Used for pacifying rooms full of people with less-than-lethal force. It warns against use on people with lung defects."
+	path = 1
+	stage = 2
+
+	/obj/item/weapon/grenade/chem_grenade/gas/New()
+		..()
+		var/obj/item/weapon/reagent_containers/glass/beaker/B1 = new(src)
+		var/obj/item/weapon/reagent_containers/glass/beaker/B2 = new(src)
+
+		B1.reagents.add_reagent("sugar", 20)
+		B1.reagents.add_reagent("potassium",20)
+		B1.reagents.add_reagent("stoxin", 10)
+		B1.reagents.add_reagent("anti_toxin", 10)
+		B2.reagents.add_reagent("chloralhydrate",40)

```

Comment:
Why chloral hydrate AND soporific?

Maybe the idea is for the chloral to put them asleep and soporific to keep them down? If so you'd need a bigger quantity of the latter. otherwise i think the soporific doesnt do much here